dashboard/app/helpers/pd/workshop_survey_results_helper.rb

```diff
@@ -427,9 +429,9 @@ def generate_facilitator_averages(summary)
     facilitator_averages = facilitators.map {|name| [name, {}]}.to_h
     facilitator_averages[:questions] = {}
 
-    # In some cases, the same question has different IDs in different forms. To get around
+    # In some cases, the same question has different question names in different forms. To get around
     # this, the QUESTIONS_FOR_FACILITATOR_AVERAGES contains a primary_id (what we use
-    # here) and optional all_ids (what is used in multiple forms)
+    # here) and optional all_ids (names used in multiple forms)
```

Comment:
Not specific to this PR, now we have a chance to re-do this facilitator survey, we should enforce a specific set of question names.